plugins/catalog-backend-module-github/src/providers/GithubOrgEntityProvider.ts

```
@@ -497,16 +500,18 @@ export class GithubOrgEntityProvider implements EntityProvider {
       throw new Error('Not initialized');
     }
 
+    const logger = this.options.logger;
     const organizationTeamTransformer =
       this.options.teamTransformer || defaultOrganizationTeamTransformer;
     const { name, html_url: url, description, slug } = event.team;
     const org = event.organization.login;
-    const { headers } = await this.credentialsProvider.getCredentials({
+    const { token } = await this.credentialsProvider.getCredentials({
```

It seems like the token should be required for this to work (as per [this](https://github.com/backstage/backstage/pull/29112#discussion_r2023136425) comment). Is there some implicit configuration we request from the adopter now (and we should write documentation for)? Should we also throw an explicit error when the token is undefined?

---

I've pushed a commit which switches back to getting headers, not tokens, like the original.